tracim/models/__init__.py

```diff
@@ -48,8 +48,19 @@ def get_tm_session(session_factory, transaction_manager):
 
     """
     dbsession = session_factory()
+    # FIXME - G.M - 02-05-2018 - Check Zope/Sqlalchemy session conf.
+    # We use both keep_session=True for zope and
+    # expire_on_commit=False for sessionmaker to keep session alive after
+    # commit ( in order  to not have trouble like #52 or
```

Comment:
@inkhey you must put a complete url on tickets if you reference them in the code. it's impossible to find the bug #52 if we do not know which project/repo it is associated to.

TODO : replace #52 by https://github.com/xxxxxx/52

Reply:
Done